### DISCUSSION TITLE
wslutilities key

### DISCUSSION THREAD

**Discussion by davidp135** [sileshn/ManjaroWSL2#9] [General]
### Requirements

- [X] Yes
- [ ] No

### WSL Version

- [X] WSL 2
- [ ] WSL 1

### Version

Windows version: 10.0.22621.1702

### Kernel Version

5.15.90.1

### Repro Steps

Install ManjaroWSL2 as per instructions up to 
`sudo pacman -Syu` in "How to setup"

### Expected Behavior

`sudo pacman -Syu` completes successfully.

### Actual Behavior

Running this command shows the error:
```
error: wslutilities: key "2D4C887EB08424F157151C493DD50AA7E055D853" is unknown
:: Import PGP key 2D4C887EB08424F157151C493DD50AA7E055D853? [Y/n]
```
Type `Y`
```
error: key "2D4C887EB08424F157151C493DD50AA7E055D853" could not be looked up remotely
error: failed to synchronize all databases (invalid or corrupted database (PGP signature))
```

**Comment by sileshn** [comment]
https://github.com/wslutilities/wslu/issues/278#issuecomment-1589870056
